src/components/common/ColonyActions/CountDownTimer/CountDownTimer.tsx

```diff
@@ -55,139 +38,30 @@ interface Props {
 
 const CountDownTimer = ({
   state,
```

Comment:
Any chance we could have all references to the MotionState as "motionState" (instead of "state")? It would maintain consistency and I find it easier to read

Reply:
Also state could refer to all sorts of things